test/lib/rule-finder.js

```diff
@@ -147,12 +170,12 @@ describe('rule-finder', function() {
     ])
   })
 
-  it('specifiedFile (absolut path) - current rules', function() {
+  it('specifiedFile (absolute path) - current rules', function() {
```

Comment:
Oh my typo 😢 , thanks for fixing them.
Did they just caught your eye, or any __editor-plugin__?